### PR TITLE
Fix prometheus after hyper 14 upgrade

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: utils/prometheus/Cargo.toml [hunk 1]
```diff
@@ -22,3 +22,7 @@ derive_more = "0.99"
 async-std = { version = "1.6.5", features = ["unstable"] }
 tokio = "1.10"
 hyper = { version = "0.14.11", default-features = false, features = ["http1", "server", "tcp"] }
+
+[dev-dependencies]
+hyper = { version = "0.14.11", features = ["client"] }
+tokio = { version = "1.10", features = [ "rt-multi-thread" ] }
```
Comment (niklasad1) [inline utils/prometheus/Cargo.toml:28]: use tokio tests (i.e. the behind --feature macros)?

File: utils/prometheus/src/lib.rs [hunk 2]
```diff
@@ -162,3 +172,48 @@ mod known_os {
...
+			&registry,
+		)
+		.expect("Registers the test metric");
+
+		runtime.spawn(init_prometheus_with_listener(listener, registry));
+
+		runtime.block_on(async {
+			let client = Client::new();
+
+			// Make a GET /ip to 'http://httpbin.org'
```
[the remark this answers is not in the document]
Comment (bkchr) [reply utils/prometheus/src/lib.rs:206]: Ahh :D

File: utils/prometheus/src/lib.rs [hunk 3]
```diff
@@ -137,18 +137,28 @@ mod known_os {
...
+		use networking::Incoming;
+
+		log::info!("〽️ Prometheus exporter started at {}", listener.local_addr()?);
 
 		let service = make_service_fn(move |_| {
 			let registry = registry.clone();
 
 			async move {
 				Ok::<_, hyper::Error>(service_fn(move |req: Request<Body>| {
+					eprintln!("{:?}", req);
```
[the remark this answers is not in the document]
Comment (bkchr) [reply utils/prometheus/src/lib.rs:161]: More ahh

File: utils/prometheus/Cargo.toml (hunk 1 above)
Comment (niklasad1) [inline utils/prometheus/Cargo.toml:28]: inconsitent whitespaces, I suppose this is preferred?

```suggestion
tokio = { version = "1.10", features = ["rt-multi-thread"] }
```

**Review by niklasad1 — APPROVED** [review]

**Review by koute — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**